### PR TITLE
Added Hyra Features to Documentation.

### PR DESCRIPTION
**Pull request by wert01930**

### CODE REVIEW

File: docs/hyra-features/activity-tracking.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 2
+---
+# Activity Tracking
+Activity Tracking is essential on many workspaces to make sure that their staff is working enough to make sure every guests is managed enough and have the best time in their experiences. Here, we'll be featuring different features that can help you manage your staff activity using Hyra's Activity Tracking and Assignments!
```
Comment (Sam512) [inline docs/hyra-features/activity-tracking.md]: "is" should be "are"

Comment (Sam512) [inline docs/hyra-features/activity-tracking.md]: Replace guests with guest

File: docs/hyra-features/activity-tracking.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 2
+---
+# Activity Tracking
+Activity Tracking is essential on many workspaces to make sure that their staff is working enough to make sure every guests is managed enough and have the best time in their experiences. Here, we'll be featuring different features that can help you manage your staff activity using Hyra's Activity Tracking and Assignments!
+
+## Activity
+In the Activity Tab, your staff will be able to see their own activity. Hyra also features a activity history that shows how active you are in the past year! Other than that, you can also see a leaderboard to show the most active staff to beat!
+
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1037567739692339320/image.png)
```
Comment (Sam512) [inline docs/hyra-features/activity-tracking.md]: Do not hotlink Discord. Store images locally in repo.

File: docs/hyra-features/activity-tracking.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Activity
+In the Activity Tab, your staff will be able to see their own activity. Hyra also features a activity history that shows how active you are in the past year! Other than that, you can also see a leaderboard to show the most active staff to beat!
+
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1037567739692339320/image.png)
+
+## Assignments
+Assignments can help you deploy tasks that your staff are needed to do. For now, it is only limited to sessions or minutes, but in the future, we may expand it to more requirements.
+
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1037568789568569344/image.png)
```
Comment (Sam512) [inline docs/hyra-features/activity-tracking.md]: Do not hotlink.

File: docs/hyra-features/dicipline-management.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+---
+# Dicipline Management
+Dicipline Management is important in any workspace. We know we have staff where they refuse to do work and disrespect the higher-ups. And this is where warnings can be helpful! Here, we'll be featuring different features that can help you manage warnings, promotions/demotions, suspensions, terminations, etc. using Hyra's Logbook and Staff Views!
```
Comment (Sam512) [inline docs/hyra-features/dicipline-management.md]: Remove "." before "And".

Comment (Sam512) [inline docs/hyra-features/dicipline-management.md]: Replace "we'll" with "we will".

File: docs/hyra-features/dicipline-management.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+---
+# Dicipline Management
+Dicipline Management is important in any workspace. We know we have staff where they refuse to do work and disrespect the higher-ups. And this is where warnings can be helpful! Here, we'll be featuring different features that can help you manage warnings, promotions/demotions, suspensions, terminations, etc. using Hyra's Logbook and Staff Views!
+
+## Logbook
+Logbook is here to help you log all of the previous actions taken and any notes about the user, this feature is especially helpful for big teams that needs to manage their staff's Warnings, Promotions/Demotions, Suspensions, Termination, etc.
+
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1026734694399741962/logbook_1.gif)
```
Comment (Sam512) [inline docs/hyra-features/dicipline-management.md]: Do not hotlink.

File: docs/hyra-features/dicipline-management.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+Logbook is here to help you log all of the previous actions taken and any notes about the user, this feature is especially helpful for big teams that needs to manage their staff's Warnings, Promotions/Demotions, Suspensions, Termination, etc.
+
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1026734694399741962/logbook_1.gif)
+
+## Staff Views
+Staff views are how workspace admins can view information about their employees all at once. Hyra's design is made to help you efficiently manage your group. In this feature, you can create customized views depending on what details are most important.
+
+You can also sort people based on their warnings recieved!
+
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1037562248547270676/Untitled_design.gif)
```
Comment (Sam512) [inline docs/hyra-features/dicipline-management.md]: Do not hotlink.

File: docs/hyra-features/dicipline-management.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+# Dicipline Management
+Dicipline Management is important in any workspace. We know we have staff where they refuse to do work and disrespect the higher-ups. And this is where warnings can be helpful! Here, we'll be featuring different features that can help you manage warnings, promotions/demotions, suspensions, terminations, etc. using Hyra's Logbook and Staff Views!
+
+## Logbook
+Logbook is here to help you log all of the previous actions taken and any notes about the user, this feature is especially helpful for big teams that needs to manage their staff's Warnings, Promotions/Demotions, Suspensions, Termination, etc.
+
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1026734694399741962/logbook_1.gif)
+
+## Staff Views
+Staff views are how workspace admins can view information about their employees all at once. Hyra's design is made to help you efficiently manage your group. In this feature, you can create customized views depending on what details are most important.
```
Comment (Sam512) [inline docs/hyra-features/dicipline-management.md]: Switch customized to customised. Hyra uses Oxford English.

File: docs/hyra-features/session-registers.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 3
+---
+# Session Registers
+Sessions are useful to book shifts and sessions that your staff can attend to! This is useful for Training Sessions to make sure there is a group of staff that are free and able to train trainees in the Training Center.
+
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1037569787049541653/image.png)
```
Comment (Sam512) [inline docs/hyra-features/session-registers.md]: Do not hotlink.

File: docs/hyra-features/session-registers.md [hunk 9]
```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 3
+---
+# Session Registers
+Sessions are useful to book shifts and sessions that your staff can attend to! This is useful for Training Sessions to make sure there is a group of staff that are free and able to train trainees in the Training Center.
```
Comment (Sam512) [inline docs/hyra-features/session-registers.md]: Remove random capital letter on "Training"

File: guides/sessions/role-groups.md [hunk 10]
```diff
@@ -5,6 +5,12 @@ sidebar_position: 2
 # Role Groups
 **Role Groups** are the foundation of attendance at sessions. They are available to be given to one staff member and are arranged into groups. For example, you might have a role group named 'Cashier A' with 1 trainer and 3 assistants beneath it.
 
+:::danger
+Please note that you **cannot** edit/delete roles if they are added into a session. You first need to remove the role group to the session before doing any changes.
+
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1036974487654109204/unknown.png)
```
Comment (Sam512) [inline guides/sessions/role-groups.md]: Do not hotlink.

File: guides/sessions/role-groups.md [hunk 11]
```diff
@@ -17,7 +23,7 @@ After giving it a name, you can now set how many roles does that group have? For
 Repeat the process until you've added all of the roles that you want.
 
 :::note
-Only **one person** can join the role. That means if you have added 4 Trainers, there will only be 4 trainers in the session.
+Please note that only **one person** can join the role. That means if you have added 4 Trainers, there will only be 4 trainers in the session.
 
 ![](https://cdn.discordapp.com/attachments/1015246364692717609/1019230377539276851/unknown.png)
```
Comment (Sam512) [inline guides/sessions/role-groups.md]: Do not hotlink.

File: guides/staff-management/staff-views.md [hunk 12]
```diff
@@ -1,5 +1,6 @@
 # Staff Views
 Staff views are how workspace admins can view information about their employees all at once. Hyra's design is made to help you efficiently manage your group. In this feature, you can create customized views depending on what details are most important.
```
Comment (Sam512) [inline guides/staff-management/staff-views.md:2]: - [ ] Swap customized for customised.

File: guides/staff-management/staff-views.md [hunk 13]
```diff
@@ -1,5 +1,6 @@
...
+
 ## How to get Started
 To start off, click the staff icon on the left side of your screen. You will see the dashboard which lists the amount of staff, the average activity for that week, and the top performing employees. Underneath the dashboard button, click **"Create a view"**.
```
Comment (Sam512) [inline guides/staff-management/staff-views.md:5]: - [ ] Replace dashboard with Dashboard

File: guides/workspace-settings/labels.md [hunk 14]
```diff
@@ -1,14 +1,25 @@
...
-Labels are great for your team's administration to organize certain people. 
+Labels are great for your team's administration to organize certain people. You can use them to label staff from their timezone, perks, or status.
 
-Some ideas may be:
-- Timezone
-- Partnerships/Development/Support/Events/Moderation Team
-- Perks
-- Up for Promotion/Demotion
+:::caution PERMISSION REQUIRED
+To proceed, please note that you need the following permissions: **Workspace Admin**
```
Comment (Sam512) [inline guides/workspace-settings/labels.md]: Workspace admin is not required to create labels, only to delete them.

File: guides/workspace-settings/labels.md [hunk 15]
```diff
@@ -1,14 +1,25 @@
...
+:::
 
-![labels](/img/labels.PNG)
+## Creating Labels
+To create a label, go to Settings > Labels and click "Add Label."
```
Comment (Sam512) [inline guides/workspace-settings/labels.md]: Labels can also be created from the add label UI

File: guides/workspace-settings/labels.md [hunk 16]
```diff
@@ -1,14 +1,25 @@
...
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1036967516381126726/unknown.png)
 
-Create your labels in your settings -> Go to the Staff tab -> Go on any view -> Open a staff profile -> On the details tab, click "+ Add Label".
+Once created, you can change it's color and name. 
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1036968286593749063/unknown.png)
```
Comment (Sam512) [inline guides/workspace-settings/labels.md]: Do not hotlink.

File: guides/workspace-settings/labels.md [hunk 17]
```diff
@@ -1,14 +1,25 @@
...
-- Perks
-- Up for Promotion/Demotion
+:::caution PERMISSION REQUIRED
+To proceed, please note that you need the following permissions: **Workspace Admin**
+:::
 
-![labels](/img/labels.PNG)
+## Creating Labels
+To create a label, go to Settings > Labels and click "Add Label."
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1036967516381126726/unknown.png)
```
Comment (Sam512) [inline guides/workspace-settings/labels.md]: Do not hotlink.

File: guides/workspace-settings/labels.md [hunk 18]
```diff
@@ -1,14 +1,25 @@
...
 
-Create your labels in your settings -> Go to the Staff tab -> Go on any view -> Open a staff profile -> On the details tab, click "+ Add Label".
+Once created, you can change it's color and name. 
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1036968286593749063/unknown.png)
 
-![label](/img/label.PNG)
\ No newline at end of file
+## Deleting Labels
+To delete a label, go to Settings > Labels and hover to the label that you want to delete and hit **X** to delete.
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1036968448888156241/unknown.png)
```
Comment (Sam512) [inline guides/workspace-settings/labels.md]: Do not hotlink.

File: guides/workspace-settings/labels.md [hunk 19]
```diff
@@ -1,14 +1,25 @@
...
+
+:::caution PERMISSION REQUIRED
+To proceed, please note that you need the following permissions: **View Other Staff**
+:::
+
+## Adding Labels
+To add a label, go to the Staff Profile and in Labels, click "Add Label"
+![](https://cdn.discordapp.com/attachments/1015246364692717609/1036969042277306398/unknown.png)
```
Comment (Sam512) [inline guides/workspace-settings/labels.md]: Do not hotlink.

**Review by Sam512 — CHANGES_REQUESTED** [review]
Please review the comments left on the files and make suitable changes.

File: guides/workspace-settings/labels.md (hunk 14 above)
Comment (wert01930) [reply guides/workspace-settings/labels.md]: Done. I've removed it.

File: guides/workspace-settings/labels.md (hunk 15 above)
Comment (wert01930) [reply guides/workspace-settings/labels.md]: ✅ Done.

File: guides/workspace-settings/labels.md (hunk 16 above)
Comment (wert01930) [reply guides/workspace-settings/labels.md]: Done. I've changed it to Hyra Drive link.

```
Once created, you can change it's color and name. 
![](https://drive.hyra.io/apps/files_sharing/publicpreview/7RFyp3DKrDD8cYJ?x=1879&y=616&a=true&file=labels-2.png&scalingup=0)
```

File: guides/workspace-settings/labels.md (hunk 17 above)
Comment (wert01930) [reply guides/workspace-settings/labels.md]: Done. Updated to Hyra Drive link.

```
## Creating Labels
To create a label, go to Settings > Labels and click "Add Label."
![](https://drive.hyra.io/apps/files_sharing/publicpreview/6BgefBRmAMKj7GN?x=1879&y=616&a=true&file=labels-1.png&scalingup=0)
```

File: guides/workspace-settings/labels.md (hunk 18 above)
Comment (wert01930) [reply guides/workspace-settings/labels.md]: Done. Updated to Hyra Drive link.

```
## Deleting Labels
To delete a label, go to Settings > Labels and hover to the label that you want to delete and hit **X** to delete.
![](https://drive.hyra.io/apps/files_sharing/publicpreview/7WfHzZ8J4PqKgjE?x=1879&y=616&a=true&file=labels-3.png&scalingup=0)
```

File: guides/workspace-settings/labels.md (hunk 19 above)
Comment (wert01930) [reply guides/workspace-settings/labels.md]: Done. Updated to Hyra Drive link.

```
## Adding Labels
To add a label, go to the Staff Profile and in Labels, click "Add Label"

![](https://drive.hyra.io/apps/files_sharing/publicpreview/2i5YwY6L4Hq7736?x=1879&y=616&a=true&file=labels-4.png&scalingup=0)
```

**Review by Sam512 — CHANGES_REQUESTED** [review]
Further changes still required

**Review by Sam512 — APPROVED** [review]
LGTM

**Pull request merged**